tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -2999,6 +2999,132 @@ var _ = ginkgo.Describe("TiDBCluster", func() {
 		})
 	})
 
+	ginkgo.Describe("Start Script Version", func() {
+
+		type testcase struct {
+			nameSuffix string
+			tlsEnable  bool
+		}
+
+		cases := []testcase{
+			{
+				nameSuffix: "",
+				tlsEnable:  false,
+			},
+			{
+				nameSuffix: "and enable TLS",
+				tlsEnable:  true,
+			},
+		}
+
+		for _, testcase := range cases {
+			ginkgo.It("deploy cluster with start script v2 "+testcase.nameSuffix, func() {
```

Comment:
We need to add similar two cases for across Kubernetes deployment.

Reply:
Done https://github.com/pingcap/tidb-operator/pull/4505/commits/42f2b0f1ba0aa938589b94a72cfdbe547545f012